site/content/entry/yuzu-progress-report-may-2022/index.md

```diff
@@ -0,0 +1,235 @@
+
+The Network tab in `Emulation > Configure… > System` could remain untranslated after changing languages. This was a simple case of forgetting to include the tab in the translations, so Docteh {{< gh-hovercard "8293" "fixed the oopsie" >}} and the lone Network tab now displays as it should.
+
+{{< single-title-imgs-compare
+    "Netto-kun. Now I want a Battle Network Legacy Collection, c'mon CAPCOM!"
+    "./netbug.png"
+    "./netfix.png"
+>}}
+
+yuzu’s icon in the About dialog has been in a struggle lately, a fix for Linux builds would break Windows support, and vice versa.
+Via qtcreator, Docteh {{< gh-hovercard "8339" "fixed the About dialog UI file," >}} and removed an old warning caused by the original .png image.
+Thanks for taking the time to properly address the issue once and for all!
+
+## Controller changes
+
+Motion continued reporting data when disabled, causing `Pokémon Let’s Go, Eevee/Pikachu!` to spam `StopSixAxisSensor` errors in the log.
+While working on this, [german77](https://github.com/german77) noticed a missing parameter, `delta_time`.
+Its implementation allows yuzu to have an {{< gh-hovercard "8308" "accurate motion refresh rate," >}} equal to the Switch.
+
+In an all-in-one pull request, german77 made {{< gh-hovercard "8368" "several input changes," >}} including:
+
+- Add proper error handling for several [HID](https://en.wikipedia.org/wiki/Human_interface_device) functions
+- Improve previous implementations to match more closely to native hardware.
+- Implement functions needed by `Nintendo Switch Sports`, `EnableSixAxisSensorUnalteredPassthrough`, `IsSixAxisSensorUnalteredPassthroughEnabled`, `LoadSixAxisSensorCalibrationParameter`, `GetSixAxisSensorIcInformation`, `ResetIsSixAxisSensorDeviceNewlyAssigned`.
+
+Nintendo Switch Sports will only be playable after an audio rewrite and more GPU work.
+
+`Arcaea` was reported as having issues with touch emulation.
+Turns out this game checks the position on release and some input drivers didn't keep it's position on release. 
+Also, multi-touch wasn't working on touch screens.
+
+After performing basically {{< gh-hovercard "8372" "a mini-rewrite of the touch emulation," >}} german77 fixed both issues.
+
+{{< imgs
+    "./arcaea.png| Osu! but better? (Arcaea)"
+  >}}
+
+One of the hurdles when working with an infinite amount of different controllers is the different quality of implementations they have.
+Since yuzu used to wait for the controller to respond after sending a vibration signal, slow controllers could stall the whole emulator, causing severe stuttering.
+To counter this, german77 {{< gh-hovercard "8374" "moved vibration to a queue in a separate thread," >}} allowing yuzu to move along with emulation while our controllers try to do their best.
+An example of why more CPU threads help more than few faster threads, there’s always some background process that could be made asynchronous.
```

Comment:
This sentence is hard to understand. Perhaps:

"This is just another example of how emulation can often be improved by moving blocking operations to asynchronous background threads to improve overall usability. In fact, yuzu uses dozens of threads for emulation, which is all the more reason why eliminating spinlocks really helps things to run as smooth as butter!"

(bet you liked that tie in to the section above, huh?)